src/test/java/testPackage01/Test_sendGraphqlRequest.java

```diff
@@ -0,0 +1,32 @@
+package testPackage01;
+
+import com.shaft.api.RestActions;
+import com.shaft.validation.Assertions;
+import io.restassured.response.Response;
+import org.testng.annotations.Test;
+
+public class Test_sendGraphqlRequest {
+    @Test
+    public void sendGraphqlRequestUsingQuery(){
+
+        String query = "{company {ceo}}";
+        Response response = RestActions.sendGraphqlRequest("https://api.spacex.land/",query);
+        Assertions.assertApiResponseEquals(response,"Elon Musk","data.company.ceo");
+    }
+    @Test
+    public void sendGraphqlRequestUsingMutationAndVariables(){
+
+        String mutation = "mutation ($name:String,$rocket:String){\n" +
```

Comment:
Since we're using java 15, please note that you can use multi-line strings to facilitate writing your mutations and variables.

the syntax is simple String mutation = """
""";

you must have a line break after the three quotation marks, and then simply paste your string on the second line. It will be much more readable and you won't need any of the "\n" or escape characters.